### DISCUSSION TITLE
VSIX version possible?

### DISCUSSION THREAD

**Discussion by F4ceMeltor** [dankeboy36/vscode-arduino-api#9] [Ideas]
It'd be great if you could pack this into a VSIX, because I'm trying to run your ESP32 exception decoder in VSCodium and that one's extension browser finds the decoder extension and can install it just fine but fails to get this one which it depends on. It'd be a great help, as I don't really know how to transfer the error properly into the Arduino IDE.

**Comment by dankeboy36** [comment]
Sure. I opened an issue: https://github.com/dankeboy36/vscode-arduino-api/issues/10.

**Comment by kittaakos** [comment]
@F4ceMeltor, once `vscode-arduino-api` is available from the VS Code Marketplace or the Open VSX Registry as a VSIX, you will overcome your current error. Still, the ESP exception decoder or any other IDE2 extension will be non-functional. It is the Arduino IDE that sets the state. So, nothing will work if you have all these VSIXs in VS Code or VSCodium but not in the Arduino IDE: https://github.com/arduino/arduino-ide/pull/2110.

**Reply by F4ceMeltor** [reply]
I'm sorry, I'm pretty new at this stuff. I'm mostly using PlatformIO in VSCodium to tool around with the ESP32, but I do have the Arduino IDE 1 and 2 installed, from a small project I did half a year ago using an Arduino. I'm just not sure how it all fits together.
